include/electionguard/ballot.hpp

```diff
@@ -134,6 +134,7 @@ namespace electionguard
 
       private:
         class Impl;
+#pragma warning(suppress : 4251)
```

Comment:
i went back and forth on whether or not to suppress this warning globally or to do it at each location.  im not sure i made the right choice.